pkg/kubelet/cm/cpumanager/policy.go

```diff
@@ -38,4 +39,6 @@ type Policy interface {
 	// and is consulted to achieve NUMA aware resource alignment per Pod
 	// among this and other resource controllers.
 	GetPodTopologyHints(s state.State, pod *v1.Pod) map[string][]topologymanager.TopologyHint
+	// GetAssignableCPUs returns the assignable (not allocated) CPUs
+	GetAssignableCPUs(m state.State) cpuset.CPUSet
```

Comment:
Let's call this `GetAllocatableCPUs()` for consistency.

Reply:
ack